solana/pyth2wormhole/program/src/attest.rs

```diff
@@ -49,14 +49,52 @@ use solitaire::{
 
 pub type P2WEmitter<'b> = Derive<Info<'b>, "p2w-emitter">;
 
+/// Important: must be manually maintained until native Solitaire
+/// variable len vector support. The number must correspond to maximum
+/// number of symbols and reflect how many pyth product/price pairs
+/// are expected in the Attest struct below.
+pub const P2W_MAX_BATCH_SIZE: u16 = 5;
+
 #[derive(FromAccounts, ToInstruction)]
 pub struct Attest<'b> {
```

Comment:
let me make sure I understand how this works. The limitation in this struct is purely a limitation of the on-chain program that we invoke to construct the attestation message, right? That is, this is something we can improve later once we figure out the stack size thing.